Mathlib/Combinatorics/Quiver/Covering.lean

```diff
@@ -139,7 +139,7 @@ theorem Prefunctor.symmetrifyStar (u : U) :
   erw [Equiv.eq_symm_comp]
   ext ⟨v, f | g⟩ <;>
     -- Porting note (https://github.com/leanprover-community/mathlib4/issues/10745): was `simp [Quiver.symmetrifyStar]`
-    simp only [Quiver.symmetrifyStar, Function.comp_apply] <;>
+    simp [Quiver.symmetrifyStar, Function.comp_apply] <;>
```

Comment:
You're introducing a non-terminal simp now. To my naive mind, that's a regression.

Reply:
Oh, I'm blind, I'll fix this one.